engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeVisualizationsTest.scala

```diff
@@ -234,13 +234,43 @@ class RuntimeVisualisationsTest
 
     object Visualisation {
 
+      val metadata = new Metadata
+
+      val code =
+        metadata.appendToCode(
+          """
+            |encode x = x.to_text
+            |
+            |incAndEncode x =
+            |    y = x + 1
+            |    encode y
+            |
+            |""".stripMargin.linesIterator.mkString("\n")
```

Comment:
```suggestion
            |""".stripMargin
```

isn't this bit spurious? This is equivalent, right?